Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM denoland/deno
+
+EXPOSE 8000
+
+WORKDIR /app
+
+ADD . /app
+
+RUN deno cache server/main.ts
+
+ARG version
+ENV DENO_DEPLOYMENT_ID $version
+CMD deno run -A server/main.ts
```

Comment:
I'd also set `USER deno:deno` somewhere so that the process doesn't run as root.

Reply:
Agreed and resolved in `e8ec58e`.